anvilPoll2024ExtraAnalysis.Rmd

```diff
@@ -1028,16 +1016,66 @@ resultsTidy %>%
 
 ```
 
-## AnVIL Support Forum Awareness
+## Awareness: AnVIL Support Forum
 
-### Raw responses
+```{r}
+forumdf %<>% mutate(,
+        forumUse = factor(
+          case_when(
+          forumInteractionDescription == "Posted in" ~ "Have utilized",
+          forumInteractionDescription == "Answered someone's post" ~ "Have utilized",
+          forumInteractionDescription == "Read through others' posts" ~ "Have utilized",
+          forumInteractionDescription == "No but aware of" ~ "Have not utilized",
+          forumInteractionDescription == "No didn't know of" ~ "Have not utilized"
+        ), levels = c("Have not utilized", "Have utilized")))
+```
 
-```{r, message=FALSE, echo=FALSE}
-forumPlotRaw <- ggplot(forumdf, aes(y = reorder(forumInteractionDescription, count), x = count, fill = UserType)) + 
+### Utilization
+
+```{r}
+forumPlotUtil <- forumdf %>%
+  group_by(UserType, forumUse) %>%
+  summarize(count = n()) %>%
+  ggplot(aes(y=reorder(forumUse, count),
+             x= count,
+             fill = UserType)) + 
   geom_bar(stat = "identity") +
   ggtitle("Have you ever read or posted in our AnVIL Support Forum?")
 
-stylize_bar(forumPlotRaw)
+stylize_bar(forumPlotUtil)
+```  
+
+### Awareness or Utilization Color rather than y-axis split
+
+```{r}
```

Comment:
Let's add a `theme(legend.title = element_blank()) +` to these two.

Might be nice to place these two side by side (either facet or patchwork). Same with the "Awareness or Utilization Color rather than y-axis split" for AnVIL Demos.